libcxx/test/std/containers/container.adaptors/flat.multimap/scary.compile.pass.cpp

```diff
@@ -0,0 +1,33 @@
+//===----------------------------------------------------------------------===//
```

Comment:
@huixie90 Can you chime in on your preferred way of checking this if we don't already?